moveit_setup_assistant/src/tools/moveit_config_data.cpp

```diff
@@ -724,6 +724,26 @@ void MoveItConfigData::outputFollowJointTrajectoryYAML(YAML::Emitter& emitter,
   }
 }
 
+// ******************************************************************************************
+// Helper function to get the default start state group for moveit_sim_hw_interface
+// ******************************************************************************************
+std::string MoveItConfigData::getDefaultStartStateGroup()
+{
+  if (!srdf_->srdf_model_->getGroups().empty())
+    return srdf_->srdf_model_->getGroups()[0].name_;
+  return "todo_no_group_selected";
```

Comment:
@MohmadAyman, what do you plan to do with this name?

Reply:
Well this name should be added to the `moveit_sim_controllers` if there was no planning groups to avoid writing blank.

Reply:
I suggest to issue a warning in MSA when no group was selected. Also, in this case, the sim_controllers shouldn't be written?

Reply:
I'm not 100% sure I remember correctly, but I think the reason we went for writing a dummy group name, was to make the output controllers file more of a template for users, so when a user is to use a `sim_controller` will just have to edit the name of the group.
also a warring sounds nice. - but I cant access my dev computer for a few days, maybe someone add it and I'll approve-